### ISSUE TITLE
Helm chart ingress issue with TLS terminated by load balancer

### ISSUE THREAD

**Issue by Chili-Man** [gravitational/teleport#5013]
### Description
I'm trying to deploy Teleport with the Helm chart.  I have the ingress-nginx chart installed as well which is behind a TLS terminating AWS Network load balancer. When I enable the ingress option on the Helm chart (`ingress.enabled`) without setting the `ingress.tls`  options because there's already a load balancer terminating the TLS,  it fails to login

**What happened**:

When I try login to via tsh, I get the following error:

```shell
$ tsh login --proxy teleport.example.com:443 --insecure --user chili-man
WARNING: You are using insecure connection to SSH proxy https://teleport.example.com:443
Enter password for Teleport user chili-man:
Enter your OTP token:
731263
WARNING: You are using insecure connection to SSH proxy https://teleport.example.com:80
error: Post "https://teleport.example.com:80/v1/webapi/ssh/certs": http: server gave HTTP response to HTTPS client
```

**What you expected to happen**:

The problem seems to because the chart sets the port for the `proxy_service` `public_addr` to `80` when the TLS options are not set: https://github.com/gravitational/teleport/blob/master/examples/chart/teleport/templates/config.yaml#L76

**How to reproduce it (as minimally and precisely as possible)**:

Create a Helm chart vaules file with the following:

```yaml
  license:
    # Use OSS version instead of Enterprise
    enabled: false

  proxy:
    tls:
      enabled: true
      usetlssecret: false

  ingress:
    enabled: true
    hosts:
       teleport.example.com:
         - /

    annotations:
      # We need to set this since teleport will be serving with a self signed
      # TLS certificate
      nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
      nginx.ingress.kubernetes.io/proxy-ssl-verify: "false"

  config:
    public_address: teleport.example.com

```

### Environment
- Teleport version (use `teleport version`): `Teleport v5.0.0 git:v5.0.0-0-gac4971801 go1.15.5`
- Tsh version (use `tsh version`): `Teleport v5.0.0 git: go1.15.5`
- OS (e.g. from `/etc/os-release`): `Amazon Linux 2`

- Where are you running Teleport? (e.g. AWS, GCP, Dedicated Hardware):
AWS EKS

**Comment by webvictim** [comment]
@Chili-Man Your sample config file above shows `ingress.enabled` as `false` but you also say above that the error occurs when `ingress.enabled` is true. Which is it?

**Comment by Chili-Man** [comment]
@webvictim  oops, it should be set to true; I've updated the example; I had copied it from my working example without the ingress enabled to bypass the issue.

**Comment by webvictim** [comment]
This has been fixed in `v0.0.8` of the Helm chart. It should be out on https://charts.releases.teleport.dev/ around midnight UTC.
